tests/utils/test_db.py

```diff
@@ -0,0 +1,23 @@
+import os
+
+import pytest
+from page_analyzer.utils.db import create_url
+
+
+@pytest.fixture()
+def set_test_environ_var():
+    os.environ[
+        "DATABASE_URL"
+    ] = "postgresql://page-analyzer:page-analyzer@localhost:5432/page-analyzer"
+    return None
+
```

Comment:
let's add a [teardown](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#yield-fixtures-recommended) for this fixture with cascade truncate of `urls` table - it will make our tests start with fresh empty DB